monkey/monkey_island/cc/ui/src/components/pages/ConfigurePage.js

```diff
@@ -50,6 +51,20 @@ class ConfigurePageComponent extends AuthComponent {
     };
   }
 
+  getSectionsOrder() { // TODO: Fetch mode from API endpoint
+    // let mode = "";
+    // this.authFetch('/api/mode')
+    //   .then(res => res.json())
+    //     .then(res => {
+    //       mode = res.mode
+    //     }
+    //   );
+
+    // let mode = 'ransomware';
+    let mode = 'advanced';
+    return CONFIGURATION_TABS_PER_MODE[mode];
```

Comment:
If the island mode is not set then the advanced tabs will be shown right ?

Reply:
I'm assuming that the endpoint will return "advanced" or "ransomware", according to the tests:
https://github.com/guardicore/monkey/blob/a77b7ea7bb2a6f5e3130d222e6ab8ae251fcff4f/monkey/tests/unit_tests/monkey_island/cc/resources/test_island_mode.py#L15